packages/shared/src/components/BookmarkFeedLayout.tsx

```diff
@@ -100,7 +100,7 @@ export default function BookmarkFeedLayout({
       </FeedPageHeader>
       <CustomFeedHeader
         className={classNames(
-          'mb-6 flex',
+          'mb-6',
```

Comment:
Why was this related? or just a bug you caught?

Reply:
Oh, sorry forgot to mention. There was some bug tracing to this element and noticed the flex was applied twice so i removed this excess one.